negy-gateway/src/main.rs

```diff
@@ -66,9 +69,12 @@ async fn fetch_nodes_unselected(
         .filter(|n| {
             if let Some(min_version) = &min_version {
                 Version::parse(&n.version).unwrap() >= Version::parse(min_version).unwrap()
-            } else {
-                true
-            }
+            } else {true}
```

Comment:
Is this change what your format does?

Reply:
I did that becuase it's a lttile more compact and just as readable.
However, I can change it back if you want.

Reply:
You shoudn't do that. We should follow rustfmt. I configure my editor as run rustfmt automatically right before I save the file.

Reply:
Okay, I'm about to update my fork here in second. Can rustfmt be run in the terminal?

Reply:
Yes. The best way to run rustfmt depends on your environment. So google it.